target/linux/x86/base-files/lib/upgrade/platform.sh

```diff
@@ -39,19 +59,42 @@ platform_check_image() {
 platform_copy_config() {
 	local partdev parttype=ext4
 
-	if export_partdevice partdev 1; then
+	if (is_efi_system && export_partdevice_label partdev openwrt_boot) \
+		|| export_partdevice partdev 1 ; then
+		mkdir -p /tmp/boot
 		part_magic_fat "/dev/$partdev" && parttype=vfat
-		mount -t $parttype -o rw,noatime "/dev/$partdev" /mnt
-		cp -af "$UPGRADE_BACKUP" "/mnt/$BACKUP_FILE"
-		umount /mnt
+		mount -t $parttype -o rw,noatime "/dev/$partdev" /tmp/boot
+		cp -af "$UPGRADE_BACKUP" "/tmp/boot/$BACKUP_FILE"
+		umount /tmp/boot
```

Comment:
Do we want to `rmdir /tmp/boot` when we're done with it?

Reply:
Hmm, [current master](https://github.com/openwrt/openwrt/blob/a662d8550fb802526dd860ad26e74b7562b34c12/target/linux/x86/base-files/lib/upgrade/platform.sh#L78) does not remove it.

It is usually created right before rebooting and it is really important data, so we'd need extra safeguards to make sure nothing else is mounted there before removing the folder, so I think it is ok as it is.